### ISSUE TITLE
[Feature request] Support configurable BertLayerNorm epsilon

### ISSUE THREAD

**Issue by huntzhan** [huggingface/pytorch-pretrained-BERT#532]
It would be great if we could configure `eps` in layer normalization since model like ERNIE uses `eps=1e-5` instead of `1e-12`.

**Comment by thomwolf** [comment]
Hi, I'm closing this in favor of #514 to gather all the discussion on ERNIE.
